src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -528,9 +528,10 @@ internal void SendFunctionLoadRequest(FunctionMetadata metadata, ManagedDependen
 
         internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, ManagedDependencyOptions managedDependencyOptions)
         {
+            var functionId = metadata.GetFunctionId();
             FunctionLoadRequest request = new FunctionLoadRequest()
             {
-                FunctionId = metadata.GetFunctionId(),
+                FunctionId = functionId,
```

Comment:
Any reason for this change? Or just style preference?

Reply:
i had done this originally for the log statement, but with that removed this became irrelevant. Going to revert this.